src/app/vault/password-generator.component.html

```diff
@@ -27,13 +27,17 @@
                         </button>
                     </div>
                     <div class="box-content condensed" [hidden]="!showOptions">
-                        <div class="box-content-row" appBoxRow>
-                            <label for="type" class="sr-only">{{'type' | i18n}}</label>
-                            <select id="type" [(ngModel)]="options.type" (change)="saveOptions()">
-                                <option value="password">{{'password' | i18n}}</option>
-                                <option value="passphrase">{{'passphrase' | i18n}}</option>
-                            </select>
-                        </div>
+                        <div class="box-content-row box-content-row-radio">
+                            <div class="radio-group text-default" appBoxRow name="PassTypeOptions"
+                                *ngFor="let o of passTypeOptions">
+                                <label for="type" class="sr-only">{{'type' | i18n}}</label>
```

Comment:
This `for` no longer refers to any element. Please delete the `for` attribute and add `radio-header` in the class. This is the pattern in the send add/edit dialog so we'll follow it here.

Reply:
Ack. I have removed the `for`. But I don't follow the radio header comment, there is already a title (OPTIONS) being set a little further up, is that sufficient?